### ISSUE TITLE
mutations_from_vcf expects VCF, not list of VCFs

### ISSUE THREAD

**Issue by pdiakumis** [umccr/umccrise#1]
* Need to change the `vcf` argument to `vcf[[1]]`
```
vcf <- read_vcfs_as_granges(file.path(params$vcf),
                            params$tumor_name,
                            ref_genome)
muts <- mutations_from_vcf(vcf)
```
As is, it gives this error:

> Warning message:
In mutations_from_vcf(vcf) :
  Some of your data is missing an ALT and/or a REF column.

**Comment by pdiakumis** [comment]
Ok, that's just an extra line lurking in the report (also in the vignette, just for exploration purposes). Variable `muts` isn't used anywhere else. Just need to delete it and close issue @vladsaveliev

**Comment by pdiakumis** [comment]
Resolved with c6cbed70d4b98a5af1195127d30ddd8e0a5e2882
